script/prove-block.sh

```diff
@@ -52,8 +52,8 @@ elif [ "$proof" == "sp1" ]; then
 	proofParam='
     "proof_type": "sp1",
 	"sp1": {
-		"recursion": "plonk",
-		"prover": "network",
+		"recursion": "core",
```

Comment:
why here using core & mock instead of plonk & network?

Reply:
well i was changing this for testing it locally since sp1 cluster was not ready then, but I guess it doesn't really matter cuz user can change this script freely